src/pages/docs/troubleshooting/salesforce-testing/most-common-issues.md

```
@@ -139,7 +139,9 @@ Users can sync metadata but encounter an MFA challenge during the process.
 
 ---
 
-## **Execution fails on Labs**
+## **Execution Fails on Labs**
+
+<img src="https://s3.amazonaws.com/static-docs.testsigma.com/new_images/projects/Updated_Doc_Images/Salesforce_Execution_Fails_on_Labs.png" style="width: 700px;" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)